mlflow/tensorflow.py

```diff
@@ -189,11 +189,9 @@ def _load_model(tf_saved_model_dir, tf_sess, tf_meta_graph_tags, tf_signature_de
             sess=tf_sess,
             tags=tf_meta_graph_tags,
             export_dir=tf_saved_model_dir)
-    # TODO: Stop relying on `tf.contrib` when it becomes deprecated. For reference, see the
-    #       Tensorflow roadmap: https://www.tensorflow.org/community/roadmap#roadmap_2.
-    signature_def = tf.contrib.saved_model.get_signature_def_by_key(
-            meta_graph_def, tf_signature_def_key)
-    return signature_def
+    if tf_signature_def_key not in meta_graph_def.signature_def:
```

Comment:
The [TF 1.12 release notes](https://github.com/tensorflow/tensorflow/blob/61c6c84964b4aec80aeace187aab8cb2c3e55a72/RELEASE.md#bug-fixes-and-other-changes) suggest that accessing `meta_graph_def.signature_def[tf_signature_def_key]` should raise a KeyError if `tf_signature_def_key` isn't in `meta_graph_def.signature_def`, but a SignatureDef is returned instead. Thus we check whether `tf_signature_def_key` is present here.